src/apps/pulse/components/App/HomeScreen.tsx

```diff
@@ -201,14 +217,117 @@ export default function HomeScreen(props: HomeScreenProps) {
   const failureGraceExpiryRef = useRef<number | null>(null);
   const hasInitializedChainIdRef = useRef<boolean>(false);
 
-  const { data: walletPortfolioData } = useGetWalletPortfolioQuery(
-    { wallet: accountAddress || '', isPnl: false },
-    {
-      skip: !accountAddress,
-      refetchOnFocus: false,
-    }
+  // Track onboarding completion in localStorage to persist across rerenders
+  const hasCompletedOnboardingRef = useRef<boolean>(
+    localStorage.getItem('hasCompletedOnboarding') === 'true'
   );
```

Comment:
Can i just check - what happens if the user uses another browser, is this a repeatable process?

Reply:
according to Aldin it should repeat when gas tank balance is less than $2 in order to top up again